writing-visualising/dashboards.qmd

```diff
@@ -513,6 +513,39 @@ An example of a basic shiny test to check that your app loads up correctly can b
 
 ---
 
+#### Migration to shinytest2
+
+---
+
+The shinytest package has been superceded by the [shinytest2 package](https://rstudio.github.io/shinytest2/). If you already have shinytest UI tests in place for your dashboard, there is a function available to migrate these tests for you so that you do not have to rewrite them all yourself. 
+
+There is [guidance on migrating to shinytest2](https://rstudio.github.io/shinytest2/articles/z-migration.html) available. To complete the migration, run `shinytest2::migrate_from_shinytest(PATH_TO_APP)`, making sure to include the full filepath. The function will search for your "tests" folder, and will then look inside the folder for any scripts containing tests. A new `testthat` folder will be automatically created, containing the code that runs your tests, as well as the snapshots generated when your tests are run. 
+
+There are a couple of potential issues that you may run into:
+
+**Chromote**
+
+Shinytest2 makes use of the Chromote package, which needs to connect to your local instance of Google Chrome. Sometimes your copy of Chrome might be installed in a different place to where Chromote expects to find it, which means that your tests will not be able to run. There is information on how to resolve this issue [in the Tips for Using R section](../learning-development/r.html#shinytest2---invalid-path-to-chrome-error-message)
```

Comment:
Scope creep a little - but in that section you refer to here we don't put anything on how to find where your chrome is installed. I wonder if someone has this issue, then it might be worth adding a quick paragraph on how to find that out? (I'm assuming there's a quick way like hit windows and right click chrome to find it or a command you could run or something to show you what path it should be?)